<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>笔记</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
    color: #333333;
}

QMenuBar {
    background-color: #f5f5f5;
    color: #333333;
    border-bottom: 1px solid #e0e0e0;
    border-radius: 4px;
}

QMenuBar::item {
    background-color: transparent;
    padding: 6px 12px;
    margin: 2px;
    border-radius: 4px;
}

QMenuBar::item:selected {
    background-color: #e0e0e0;
    color: #333333;
}

QMenu {
    background-color: #ffffff;
    color: #333333;
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    padding: 4px;
}

QMenu::item {
    padding: 6px 24px 6px 12px;
    border-radius: 4px;
    margin: 2px;
}

QMenu::item:selected {
    background-color: #e0e0e0;
}

QStatusBar {
    background-color: #f5f5f5;
    color: #666666;
    border-top: 1px solid #e0e0e0;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
    background-color: #ffffff;
    color: #333333;
    border: 1px solid #e0e0e0;
    border-radius: 12px;
    padding: 10px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 13px;
    selection-background-color: #4a90e2;
    selection-color: #ffffff;
    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
}

QTextEdit:hover {
    border: 1px solid #4a90e2;
}

QTextEdit:focus {
    border: 1px solid #4a90e2;
    box-shadow: 0px 2px 8px rgba(74, 144, 226, 0.2);
}</string>
      </property>
      <property name="placeholderText">
       <string>在此输入文字内容...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>笔记</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionInsertImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actionFontDialog"/>
    <addaction name="actionFontFamily"/>
    <addaction name="actionFontSize"/>
    <addaction name="separator"/>
    <addaction name="actionTextColor"/>
    <addaction name="actionTextHighlight"/>
    <addaction name="separator"/>
    <addaction name="actionAlignLeft"/>
    <addaction name="actionAlignCenter"/>
    <addaction name="actionAlignRight"/>
    <addaction name="actionAlignJustify"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
    background-color: #f5f5f5;
    border-bottom: 1px solid #e0e0e0;
    spacing: 6px;
    padding: 4px;
}

QToolBar::separator {
    width: 1px;
    background-color: #e0e0e0;
    margin: 6px 10px;
}

QToolButton {
    background-color: #f5f5f5;
    border: 1px solid transparent;
    border-radius: 8px;
    color: #333333;
    padding: 6px;
    margin: 2px;
    min-width: 32px;
    min-height: 32px;
}

QToolButton:hover {
    background-color: #e0e0e0;
    border: 1px solid #d0d0d0;
}

QToolButton:pressed {
    background-color: #d0d0d0;
}

QToolButton:checked {
    background-color: #e6f0fa;
    border: 1px solid #4a90e2;
}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFontDialog"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionInsertImage">
   <property name="text">
    <string>插入图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/icons/bold.svg</normaloff>:/imgs/icons/bold.svg</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/icons/italic.svg</normaloff>:/imgs/icons/italic.svg</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset>
     <normaloff>:/imgs/icons/underline.svg</normaloff>:/imgs/icons/underline.svg</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFontDialog">
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFontFamily">
   <property name="text">
    <string>字体族</string>
   </property>
   <property name="toolTip">
    <string>选择字体族</string>
   </property>
  </action>
  <action name="actionFontSize">
   <property name="text">
    <string>字号</string>
   </property>
   <property name="toolTip">
    <string>选择字号</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="text">
    <string>文本颜色</string>
   </property>
   <property name="toolTip">
    <string>设置文本颜色</string>
   </property>
  </action>
  <action name="actionTextHighlight">
   <property name="text">
    <string>文本高亮</string>
   </property>
   <property name="toolTip">
    <string>设置文本背景色</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="toolTip">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="text">
    <string>居中对齐</string>
   </property>
   <property name="toolTip">
    <string>居中对齐</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="toolTip">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="text">
    <string>两端对齐</string>
   </property>
   <property name="toolTip">
    <string>两端对齐</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
